### PR TITLE
Added 1.13 bioms to 1.12, 1.11, 1.10, 1.9 and 1.8 bioms

### PR DESCRIPTION
**Pull request by IceTank**
This should fix the missing bioms for 1.12 1.11 and 1.10 mentioned here #386

### CODE REVIEW

File: data/pc/1.10/biomes.json [hunk 1]
```diff
@@ -21,7 +21,8 @@
     "dimension": "overworld",
     "displayName": "Plains",
     "color": 9286496,
-    "rainfall": 0.4
+    "rainfall": 0.4,
+    "child": 129
```
Comment (IceTank) [inline data/pc/1.10/biomes.json:25]: Can there be more then one child for biom variations?

Comment (Karang) [reply data/pc/1.10/biomes.json:25]: No only one

**Comment by rom1504** [comment]
do you want to add the other versions too in this PR ?

**Comment by IceTank** [comment]
I added the missing bioms for 1.8 and 1.9. Burger did not extract bioms for 1.7. I have to see how I can get bioms for that version. 
I noticed that the biom display names in the 1.9 Burger extraction are not consistent. For instance there are bioms named like this: `JungleHills` and sometimes like this `Mesa Plateau`. In 1.13 they are all seperated by spaces. I included the 1.9 biom display Names.

Here is a not off how the data has been gathered:

```
1.7 No biom data from Burger 1.7

1.8 Data taken from:
name:           Burger 1.9 by id match
displayName:    Burger 1.8
id:             Burger 1.8
category:       Burger 1.13 by id match
temperature:    Burger 1.8
precipitation:  Burger 1.13 by id match
depth:          Burger 1.13 by id match
dimension:      Burger 1.13 by id match
color:          Burger 1.13 by id match
rainfall:       Burger 1.8

1.9 Data taken from:
id:             Burger 1.9
name:           Burger 1.9
category:       Burger 1.13 by id match
temperature:    Burger 1.9
precipitation:  Burger 1.13 by id match
depth:          Burger 1.13 by id match
dimension:      Burger 1.13 by id match
displayName:    Burger 1.9
color:          Burger 1.13 by id match
rainfall:       Burger 1.9
```

**Review by Karang — APPROVED** [review]
LGTM, is it still WIP ?

**Comment by IceTank** [comment]
Removed the WIP Tag. I dont know if there are any bioms missing from <=1.7. But 1.8 to 1.12 bioms should all be complet now.

**Comment by IceTank** [comment]
I dont know why 1.8 and 1.9 are different now. Some of the names and values changed. And some data is missing from the Burger extraction.

**Comment by IceTank** [comment]
The last commit has the wrong version in the commit message. It should be 1.8

**Comment by IceTank** [comment]
1.8 - 1.12 Should now have every biom with correct biome data.

**Review by rom1504 — APPROVED** [review]

**Pull request merged**